cloud-common/src/main/java/com/sequenceiq/cloudbreak/common/service/DefaultCostTaggingService.java

```diff
@@ -75,26 +79,36 @@
         return result;
     }
 
-    public Map<String, String> prepareDefaultTags(String cbUser, Map<String, String> sourceMap, String platform) {
+    public Map<String, String> prepareDefaultTags(String cbUser, Map<String, String> sourceMap, String platform, String environmentName) {
         Map<String, String> result = new HashMap<>();
         result.put(transform(CDP_USER_NAME.key(), platform), transform(cbUser, platform));
         result.put(transform(CDP_CB_VERSION.key(), platform), transform(cbVersion, platform));
+        addEnvironmentNameToMapIfValueIsNotNullOrEmpty(result, environmentName, platform);
         if (sourceMap == null || Strings.isNullOrEmpty(sourceMap.get(transform(OWNER.key(), platform)))) {
             result.put(transform(OWNER.key(), platform), transform(cbUser, platform));
         }
         result.put(transform(CDP_CREATION_TIMESTAMP.key(), platform), transform(String.valueOf(clock.getCurrentInstant().getEpochSecond()), platform));
         return result;
     }
 
-    private String transform(String value, String platform) {
+    public void addEnvironmentNameToMapIfValueIsNotNullOrEmpty(Map<String, String> hereTo, String environmentName, String platform) {
+        if (StringUtils.isNotEmpty(environmentName)) {
+            hereTo.put(transform(ENVIRONMENT_NAME.key(), platform), environmentName);
+        } else {
+            LOGGER.info("Unable to add EnvironmentName - cost - tag to the resource's default tags because it's value is empty or null!");
```

Comment:
if we really need this log, it should be debug or trace

Reply:
since it's cost-related warn, I think it should be on the info level

Reply:
@doktoric @topolyai5 what do you think?